modules/javafx.graphics/src/main/native-glass/mac/GlassWindow.m

```diff
@@ -1161,10 +1162,7 @@ static jlong _createWindowCommonDo(JNIEnv *env, jobject jWindow, jlong jOwnerPtr
     GLASS_POOL_ENTER;
     {
         GlassWindow *window = getGlassWindow(env, jPtr);
-        if (window->isResizable != jResizable)
-        {
-            [window performSelectorOnMainThread:@selector(_setResizable) withObject:nil waitUntilDone:YES];
-        }
+        [window _setResizable:jResizable];
```

Comment:
I presume you removed the `performSelectorOnMainThread` because the calling method always calls this on the main thread?

Reply:
Also, the calling Java code does a thread check. So we're good.